client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/useExposuresAndFlights.ts

```diff
@@ -129,6 +124,40 @@ export const useExposuresAndFlights = (props : Props) => {
         });
     };
 
+    const onExposureDeleted = async (index: number) => {  
+        const updatedExpousres = [...exposureAndFlightsData.exposures];
+        const deletingExposureLogger = logger.setup('Deleting Exposure');
+        alertWarning((updatedExpousres[index].wasAbroad ? flightDeleteWarningTitle : exposureDeleteWarningTitle),{
+            showCancelButton: true,
+            cancelButtonText: 'בטל',
+            cancelButtonColor: theme.palette.error.main,
+            confirmButtonColor: theme.palette.primary.main,
+            confirmButtonText: 'כן, המשך',
+        }).then((result) => {
+            if (result.value) {
+                deletingExposureLogger.info('launching exposure delete request', Severity.LOW);
+                setIsLoading(true);
+                if (updatedExpousres[index].id){
+                    const exposureId = updatedExpousres[index].id;
```

Comment:
moved updatedExposures[index] to a const, same for line 130 or do something like `const {wasAbroad , id} = upadtedExposures[index]`

Reply:
👍🏼